content/docs/start/data-and-model-access.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: 'Get Started: Data Access'
+title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data Access
+# Get Started: Data and Model Access
 
-Okay, now that we've learned how to _track_ data and models with DVC and how to
-version them with Git, next question is how can we _use_ these artifacts outside
-of the project? How do I download a model to deploy it? How do I download a
-specific version of a model? How do I reuse datasets across different projects?
+We've learned how to _track_ data files in DVC and how to commit their versions
+to Git. Machine learning models are typically large files written and read by
+programs. DVC can track and version model files similar to data files. The next
+questions are: How can we _use_ these artifacts outside of the project? How do I
+download a model to deploy it? How do I download a specific version of a model?
+How do I reuse datasets across different projects?
```

Comment:
This change seems to break the semantics of the paragraph. TBH I don't think it's necessary, we already state "data and models".

I do like the correction to the first sentence, "... , and how to commit their versions to Git" but the next 2 sentences are out of context here.